src/components/Game/Game.js

```diff
@@ -1,14 +1,129 @@
-import React from 'react';
-import classes from './Game.scss';
+import React, { Component } from 'react';
 
+import classes from './Game.scss';
 import { Slots } from './Slots/Slots';
 import { Letters } from './Letters/Letters';
 
-const Game = () => (
-    <main className={classes.Game}>
-        <Slots />
-        <Letters />
-    </main>
-);
+import { words } from '../../data/data';
+
+import { WORD_LENGTH, WORDS_COUNT } from '../../constants/constants';
+import { shuffleArray } from '../../helpers/shuffle-array';
+import { generateID } from '../../helpers/generate-id';
+import { getRandomIndexInRange } from '../../helpers/random-index';
+
+
+class Game extends Component {
+    state = {
+        correctWords: [],
+        letters: [],
+        lettersInSlots: [],
+        gameWon: null,
+    };
+
+    componentDidMount() {
+        const index = getRandomIndexInRange(WORDS_COUNT);
+        const word = words[index][0];
```

Comment:
maybe cleaner to have `const correctWords = words[index]`
then `correctWords[0].split('')` and `this.setState({ correctWords, ... })`
just a minor code style comment :)